src/apphosting/app.ts

```diff
@@ -34,22 +32,7 @@ async function getOrCreateWebApp(
   backendId: string,
 ): Promise<FirebaseWebApp | undefined> {
   const webAppsInProject = await listFirebaseApps(projectId, AppPlatform.WEB);
-
-  if (webAppsInProject.length === 0) {
-    // create a web app using backend id
-    const { displayName, appId } = await createFirebaseWebApp(projectId, {
-      displayName: backendId,
-    });
-    return { name: displayName, id: appId };
-  }
-
-  const existingUserProjectWebApps = new Map(
-    webAppsInProject.map((obj) => [
-      // displayName can be null, use app id instead if so. Example - displayName: "mathusan-web-app", appId: "1:461896338144:web:426291191cccce65fede85"
-      obj.displayName ?? obj.appId,
-      obj.appId,
-    ]),
-  );
+  const existingUserProjectWebApps = firebaseAppsToMap(webAppsInProject);
```

Comment:
Should we update the method name and comment?

Reply:
hmm i left the the name as is because it still gets the webapp if the user provides the web app via a flag. Wondering if we should remove this too.